lsp/nls/tests/inputs/completion-basic.ncl

```diff
@@ -60,7 +60,7 @@ in
 ### [[request]]
 ### type = "Completion"
 ### textDocument.uri = "file:///completion-basic.ncl"
-### position = { line = 12, character = 30 }
+### position = { line = 12, character = 26 }
```

Comment:
are we just changing what these test are testing for here? should we add new tests rather than replacing these ones, or are these ones not serving a useful purpose?

Reply:
The change shouldn't have an effect: it's moving the cursor test location from `(import "included.ncl").lala‸` to `(import "included.ncl").‸lala` to check that off-by-one behavior. One could argue that we should test both, but most of the test cases have the cursor at the end to that case is pretty well covered already...